.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  spotless:
+    name: Spotless
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Spotless
+        run: ./gradlew spotlessCheck
+
+  tests:
+    name: Tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
```

Comment:
This doesn't look quite right ;)

I'm not sure what we're actually running against here, but I assume it must be whatever Java is available in the default environment. I think there should be at least a Java setup setup somewhere?

Reply:
🤦‍♂ Nice catch.